nengo/tests/conftest.py

```diff
@@ -119,6 +120,15 @@ def analytics(request):
     return analytics.__enter__()
 
 
+@pytest.fixture
+def analytics_data(request):
+    paths = request.config.getvalue('compare')
+    function_name = parametrize_function_name(request, re.sub(
+        '^test_[a-zA-Z0-9]*_', 'test_', request.function.__name__, count=1))
```

Comment:
Yes, the replacement works that way. Not sure whether we want to keep it general.